### DISCUSSION TITLE
how to stop

### DISCUSSION THREAD

**Discussion by Scindarion** [sandarshsridhar/wiz-spotify-connected#27] [General]
how do i stop the server once its running i cant seem to get it to stop

as im getting an error saying that the client id is wrong even tho it is correct

**Comment by sandarshsridhar** [comment]
You can abort the session by calling http://localhost:8888/dance-to-spotify/abort. If you meant stopping the app altogether, you can type Ctrl+C (Windows)/ Cmd+C (MacOS).
